Stack/Opc.Ua.Core/Types/BuiltIn/NodeId.cs

```diff
@@ -799,7 +799,7 @@ public static ExpandedNodeId ToExpandedNodeId(NodeId nodeId, NamespaceTable name
 
             ExpandedNodeId expandedId = new ExpandedNodeId(nodeId);
 
-            if (nodeId.NamespaceIndex > 1)
+            if (nodeId.NamespaceIndex > 0)
```

Comment:
@AlinMoldovean , I believe this is an optimization because index 1 is always the namespace of the local server and ExpandedNodeId is used in many places as a coding convenience in stead of NodeId, so there is really no point to convert index 1.
Actually the conversion could cause quite a perf hit. Could you double check there is no perf impact and the change is really fixing an issue?

Reply:
You miss the case on the a client side. 
Sample: server has such namespace and client is connected to the server using an assembly with corresponding encodable types (structures). Such types are added into the session's EncodeableFactory. The client trying to call some method of the server that returns an instance of such encodable type. The Client got ExtensionObject as method result and can't decode its content because of this bug. This is a real case. This is real bug that blocks communication between client and server.